src/ciphersuite.rs

```diff
@@ -24,7 +23,7 @@ pub trait CipherSuite {
     /// an extension trait PasswordToCurve that allows some customization on
     /// how to hash a password to a curve point. See `group::Group` and
     /// `map_to_curve::GroupWithMapToCurve`.
-    type Group: GroupWithMapToCurve<UniformBytesLen = <Self::Hash as Digest>::OutputSize>;
```

Comment:
I hope removing this bound is fine, I guess when we merge both trait's it gonna be unnecessary anyway.